spec/src/main/asciidoc/changelog.adoc

```diff
@@ -24,6 +24,12 @@
 
 Changes marked with icon:bolt[role="red"] are breaking changes relative to previous versions of the spec.
 
+[[release_notes_2_4]]
+== Changes in 2.4
```

Comment:
I titled it 2.4 instead of 3.0 because we're not yet absolutely sure that we'll need a major bump. Let's update that to 3.0 once we actually have a breaking change